x-pack/test_serverless/functional/page_objects/svl_search_inference_management_page.ts

```diff
@@ -0,0 +1,114 @@
+        await testSubjects.existOrFail('view-your-models');
+      },
+
+      async expectTabularViewToBeLoaded() {
+        await testSubjects.existOrFail('search-field-endpoints');
+        await testSubjects.existOrFail('type-field-endpoints');
+        await testSubjects.existOrFail('service-field-endpoints');
+
+        const table = await testSubjects.find('inferenceEndpointTable');
+        const rows = await table.findAllByClassName('euiTableRow');
+        expect(rows.length).to.equal(1);
+
+        const endpointCell = await rows[0].findByTestSubject('endpointCell');
+        const endpointName = await endpointCell.getVisibleText();
+        expect(endpointName).to.contain('endpoint-1');
+
+        const providerCell = await rows[0].findByTestSubject('providerCell');
+        const providerName = await providerCell.getVisibleText();
+        expect(providerName).to.contain('Elasticsearch');
+        expect(providerName).to.contain('.elser_model_2');
+
+        const typeCell = await rows[0].findByTestSubject('typeCell');
+        const typeName = await typeCell.getVisibleText();
+        expect(typeName).to.contain('sparse_embedding');
+      },
+
+      async expectEndpointWithoutUsageTobeDelete() {
+        await testSubjects.click('inferenceUIDeleteAction');
+        await testSubjects.existOrFail('deleteModalForInferenceUI');
+        await testSubjects.existOrFail('deleteModalInferenceEndpointName');
+
+        await testSubjects.click('confirmModalConfirmButton');
+
+        await retry.waitForWithTimeout('delete modal to disappear', 5000, () =>
+          testSubjects
+            .missingOrFail('confirmModalConfirmButton')
+            .then(() => true)
+            .catch(() => false)
+        );
+
+        await testSubjects.existOrFail('createFirstInferenceEndpointDescription');
+      },
+
+      async expectEndpointWithUsageTobeDelete() {
+        await testSubjects.click('inferenceUIDeleteAction');
+        await testSubjects.existOrFail('deleteModalForInferenceUI');
+        await testSubjects.existOrFail('deleteModalInferenceEndpointName');
+
+        const items = await testSubjects.findAll('usageItem');
+        expect(items.length).to.equal(2);
+
+        const index = await items[0].getVisibleText();
+        const pipeline = await items[1].getVisibleText();
+
+        expect(index.includes('elser_index')).to.be(true);
+        expect(pipeline.includes('endpoint-1')).to.be(true);
+
+        expect(await testSubjects.isEnabled('confirmModalConfirmButton')).to.be(false);
+
+        await testSubjects.click('warningCheckbox');
+
+        expect(await testSubjects.isEnabled('confirmModalConfirmButton')).to.be(true);
+        await testSubjects.click('confirmModalConfirmButton');
+
+        await retry.waitForWithTimeout('delete modal to disappear', 5000, () =>
+          testSubjects
+            .missingOrFail('confirmModalConfirmButton')
+            .then(() => true)
+            .catch(() => false)
+        );
```

Comment:
what are we doing here?

Reply:
The initial goal was to verify that the modal was missing however this is a redundant check. These were removed from both tests.